getting-started/2/README.md

```diff
@@ -165,17 +166,16 @@ Now, replace the uid's with the ones you copied, and execute the mutation.
 ![Edge between nodes](./images/edge-between-nodes.gif)
  
 ## Traversing the edges
-Graph databases offer many distinct capabilities. 
-`Traversals` are among them.
-Traversal queries start their operation from a set of root nodes. 
-These root nodes are selected using Dgraph's built-in functions.
-Then, they follow the edges from these root nodes and return the nodes at the other end of the edges.
+Graph databases offer many distinct capabilities. `Traversals` are among them.
+Traversal queries start their operation from a set of root nodes. These root nodes
```

Comment:
I would like the merge `line 1` and `line 2`, would like to retain the rest of the breakdown.